gradle-plugin/src/main/kotlin/org/mozilla/android/ext/Runtime.kt

```diff
@@ -12,13 +13,26 @@ import java.lang.IllegalStateException
  *
  * @throws IllegalStateException when the command exits with a non-zero value
  */
-internal fun Runtime.execWaitForStdOut(cmd: String): String {
+internal fun Runtime.execWaitForStdOut(cmd: String, env: Array<String>? = null, dir: File? = null): String {
     fun Process.assertNoError() {
         // We could check standard error too, but this is probably good enough.
         if (exitValue() != 0) throw IllegalStateException("Runtime exited with non-zero value: ${exitValue()}")
     }
 
-    return exec(cmd).let { process ->
+    return exec(cmd, env, dir).let { process ->
```

Comment:
nit: I don't love the duplication here. If you've got time, it'd be great to abstract the overlap into helper functions.